students/14th/juminkong/login.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width">
+    <title>Instagram</title>
+    <link href="login.css" rel="stylesheet" type="text/css" />
+    <link href="common.css" rel="stylesheet" type="text/css" />
+  </head>
+
+
+  <body>
+        <div class="container">
+        <img src= "logo_text.png" alt="instagram_logo">
+        <input type="text" class="input_ID" placeholder="전화번호, 사용자 이름 또는 이메일">
+        <input type="password" class="input_PW" placeholder="비밀번호">
+        <button type="button" class="btn">
+        <span>로그인</span>
+        </button>
+        <span class="forget"><a href="https://www.instagram.com/accounts/password/reset/">비밀번호를 잊으셨나요?</a></span>
+        </div>
```

Comment:
들여쓰기 지켜주세요!! 가독성을 위해 들여쓰기 매우 중요합니다 :) 
더불어 들여쓰기는 4tab으로 하면 코드가 너무 가로로 길어지기 때문에 2tab이 컨벤션입니다. 

```suggestion
        <div class="container">
          <img src= "logo_text.png" alt="instagram_logo">
          <input type="text" class="input_ID" placeholder="전화번호, 사용자 이름 또는 이메일">
          <input type="password" class="input_PW" placeholder="비밀번호">
          <button type="button" class="btn">
            <span>로그인</span>
          </button>
          <span class="forget">
            <a href="https://www.instagram.com/accounts/password/reset/">비밀번호를 잊으셨나요?</a>
          </span>
        </div>
```